demos/2_Iterative/03_PDHG.ipynb

```diff
@@ -301,6 +298,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Notice that islicer keeps a reference to the 3D data, therefore\n",
+    "# deleting the data3D as will happen below will not allow the garbage \n",
+    "# collector to clear the memory.\n",
```

Comment:
Is there any point in including this del data3D line then? If it's not clearing the memory can we take it out of this notebook and open an issue relating to it instead?

Reply:
Instead could have a warning not to run this cell if you are tight on memory